thirdparty/fetch-thirdparty-deps-windows.sh

```diff
@@ -16,10 +16,9 @@ fi
 
 if [ ! -f windows/freetype6.dll ]; then
 	echo "Fetching FreeType2 from nuget"
-	nuget install freetype2.redist -Version 2.4.11.3
-	cp ./freetype2.redist.2.4.11.3/bin/win32/zlib1.dll ./windows/
-	cp ./freetype2.redist.2.4.11.3/bin/win32/freetype6.dll ./windows/
-	rm -rf freetype2.redist.2.4.11.3
+	nuget install SharpFont.Dependencies -Version 2.5.5.1
```

Comment:
Can you add `-ExcludeVersion` here and below? See https://github.com/OpenRA/OpenRA/pull/7907. It is also the reason this needs a rebase now. Sorry.